### PR TITLE
Dual Zone Air Temp Sensor

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
Hi @cstirdivant - please see dual zone air temperature sensor with new abstract type for DX MDF/IDF device

### CODE REVIEW

**Comment by cstirdivant** [comment]
@Ciaran-Oflannagain  please fix validation error - looks like the canonical types name is case-sensitive

**Comment by Ciaran-Oflannagain** [comment]
@cstirdivant - thanks missed that, typo fixed

**Comment by Ciaran-Oflannagain** [comment]
hi @cstirdivant - just following up on the above PR. Many thanks, Ciaran

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml [hunk 1]
```diff
@@ -16,7 +16,15 @@
...
 ########################
 
-
+SENSOR_SS_ZTM2X:
+  guid: "fbb1219d-e7b4-4bb9-93b4-c41b5c393b65"
+  description: "Dual zone air temperature sensor."
+  is_canonical: true
+  implements:
+  - /SENSOR # Inherits from global namespace
+  - SS
```
Comment (ccquigley) [inline ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:25]: Looking for some more information on why the sensor needs SS. Is this actually a fan or is this a sensor? If it is a sensor does it actually have a start/stop or does it read values continuously?

**Review by ccquigley — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:25]: @ccquigley - thank you, apologies the BMS team have agreed to model the zone temp monitoring separately into 2 units. Closing PR

**Pull request closed without merging**